### DISCUSSION TITLE
Nextcloud is not reachable anymore

### DISCUSSION THREAD

**Discussion by EpicJosch** [nextcloud/all-in-one#4427] [Questions] [answered]
I added the community container "caddy" and then removed it again. Now the nextcloud is not reachable anymore. ERROR: ERR_QUIC_PROTOCOL_ERROR

### Steps to reproduce
1. Add caddy community container
2. Remove it again
3. Try to open nextcloud

### Expected behavior 
Nextcloud should open

### Actual behavior 
Nextcloud does not open

### Host OS 
Linux

#### Nextcloud AIO version 
Nextcloud AIO v8.0.0

#### Current channel 
stable

#### Other valuable info 
![grafik](https://github.com/nextcloud/all-in-one/assets/50040609/27573046-aead-4483-b97e-ed964c7f1635)

**Comment by szaimen** [comment, marked as answer]
Hi, this is expected because the community container changes some internal settings for easy setup: https://github.com/nextcloud/all-in-one/blob/a6f2a3c76de58be156d4f4b15855b6955e7537df/community-containers/caddy/caddy.json#L41-L42

You can fix this by restoring the backup that you created before adding the community container or setting the apache_port back to 443 and apache_ip_binding back to 0.0.0.0. see then envs: https://github.com/nextcloud/all-in-one/blob/a6f2a3c76de58be156d4f4b15855b6955e7537df/compose.yaml#L16-L17

**Comment by EpicJosch** [comment]
Thanks a lot for the help!
